patches/app-builder-lib+22.14.5.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/node_modules/app-builder-lib/out/winPackager.js b/node_modules/app-builder-lib/out/winPackager.js
```

Comment:
Can we send this patch to upstream as a pull request?

https://github.com/electron-userland/electron-builder

Reply:
yes, that is planned next step

Reply:
Yes, I am going to create a PR to upstream https://github.com/electron-userland/electron-builder with those changes.